server/settings.py

```diff
@@ -197,9 +197,10 @@
 
 
 # Files in this directory will be served by WhiteNoise at the site root.
-WHITENOISE_ROOT = os.path.join(ROOT, "build")
-STATIC_ROOT = os.path.join(ROOT, "static")
+WHITENOISE_ROOT = os.path.join(BASE_DIR, "build")
+STATIC_ROOT = os.path.join(BASE_DIR, "server")
```

Comment:
I think this line should be reverted, we want the static assets to be in `static/`, not in `server/` (which is where the application files live)

Reply:
ooh, so this is a folder at the root level, not in the server folder. i think i had been confusing these the whole time